Source/WebCore/accessibility/AccessibilityScrollView.cpp

```diff
@@ -147,6 +147,16 @@ void AccessibilityScrollView::updateScrollbars()
     if (!scrollView)
         return;
 
+    auto* localWebArea = webArea();
```

Comment:
Same question about "local" in the name.

Reply:
I am using local here to make it clear that we are referring to the object's parent web area, rather than the root one (since is many places we use the root). But, since we are in a scroll area here, we can probably get away without that.